src/kind.ts

```diff
@@ -0,0 +1,74 @@
+import { Job } from "@brigadecore/brigadier";
+
+export const kindJobImage = "radumatei/golang-kind:1.11-0.4";
+
+export class KindJob extends Job {
+    constructor(name: string, image?: string) {
+        if (image == undefined) {
+            image = kindJobImage;
+        }
+        super(name, image);
+
+        // kind needs to run as a privileged pod
+        this.privileged = true;
```

Comment:
What will the behavior be if `allowPrivilegedJobs` is set to false at the project-level on a project's `brigade.js` file attempting to use this KindJob?

Reply:
You can't run DinD, meaning you can't start the nodes as containers.